pipeline-maven/pom.xml

```diff
@@ -49,6 +49,10 @@
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>commons-lang3-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>coverage</artifactId>
+    </dependency>
```

Comment:
This is not an optional dependency but the readme implies it should be

Reply:
Fixed in https://github.com/jenkinsci/pipeline-maven-plugin/pull/931